### PR TITLE
[CI] Migrate pipeline to Blossom

### PR DESCRIPTION
**Pull request by NirWolfer**
## Description
Changes related to migration of CI to Blossom Jenkins

##### What
Change proj_jjb.yaml and job_matrix.yaml to fit the new nbu jenkins master, to allow it to run on the new jenkins instance Remove tap archives from job matrix, as it is not in use by anyone and the plugin is not installed on the new Blossom master

##### Why ?
Issue: HPCINFRA-2277

##### How ?
_It is optional but for complex PRs please provide information about the design,
architecture, approach, etc._

## Change type
What kind of change does this PR introduce?
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] CI related changes
- [ ] Documentation content changes
- [ ] Tests
- [ ] Other

## Check list
- [ ] Code follows the style de facto guidelines of this project
- [ ] Comments have been inserted in hard to understand places
- [ ] Documentation has been updated (if necessary)
- [ ] Test has been added (if possible)

### CODE REVIEW

File: .ci/redmine_jjb.yaml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+    wrappers:
+        - workspace-cleanup
+        - build-user-vars
+        - timestamps
+    
+    triggers:
+        - github-pull-request:
+            cron: 'H/5 * * * *'
+            trigger-phrase: '.*\bbot:retest\b.*'
```
Comment (dpressle) [inline .ci/redmine_jjb.yaml:31]: lets add option to run only this job (like we have in RIvermax):
'.*\bbot:(?:rm:)?retest\b.*'

**Review by dpressle — CHANGES_REQUESTED** [review]

File: .ci/redmine_jjb.yaml (hunk 1 above)
Comment (NirWolfer) [reply .ci/redmine_jjb.yaml:31]: good idea, will add

**Review by dpressle — APPROVED** [review]

**Comment by dpressle** [comment]
bot:retest

**Comment by dpressle** [comment]
bot:retest

**Pull request merged**